nextspeaker-web/src/main.rs

```diff
@@ -113,6 +162,9 @@ impl Component for Model {
                 self.candidates = Some(v);
                 self.save();
             }
+            Msg::CandidatesView => {
+                self.mode = Mode::CandidatesView;
```

Comment:
Why not model this as `Msg::changeMode(Mode)` and change the implementation to something like:

```
Msg::changeMode(m) => {
  self.mode = m;
}
```

Still obviously correct (and exhaustively checked!) and a lot less verbose, yeah?

Reply:
Yes, that's how it was in other web apps I made.  I've updated this one to do it that way, too.  Thanks.